Apps/phcrowdstrikeoauth/parse_cs_events.py

```diff
@@ -47,7 +47,7 @@ def _get_value(in_dict, in_key, def_val=None, strip_it=True):
         return def_val
 
     try:
-        if (type(in_dict[in_key]) != str) and (type(in_dict[in_key]) != unicode):
```

Comment:
Using `isinstance` is generally better. e.g.

```
if not isinstance(in_dict[in_key], str)
```

Reply:
I have made the suggested code change.